dc-cudami-server/dc-cudami-server-webapp/src/main/java/de/digitalcollections/cudami/server/controller/identifiable/entity/CollectionController.java

```diff
@@ -327,4 +366,101 @@ public ResponseEntity saveDigitalObjects(
     }
     return new ResponseEntity<>(successful, HttpStatus.NOT_FOUND);
   }
+
+  @ApiMethod(description = "Add an existing collection to an existing collection")
+  @PatchMapping(
+      value = {
+        "/latest/collections/{parentUuid}/child/{childUuid}",
+        "/v3/collections/{parentUuid}/child/{childUuid}"
+      },
+      produces = MediaType.APPLICATION_JSON_VALUE)
+  @ApiResponseObject
+  public ResponseEntity addChild(
+      @ApiPathParam(description = "UUID of the parent collection") @PathVariable("parentUuid")
+          UUID parentUuid,
+      @ApiPathParam(description = "UUID of the child collection") @PathVariable("childUuid")
+          UUID childUuid) {
+    Collection parent = new CollectionImpl();
+    parent.setUuid(parentUuid);
+
+    Collection child = new CollectionImpl();
+    child.setUuid(childUuid);
+
+    boolean successful = collectionService.addChild(parent, child);
+
+    if (successful) {
+      return new ResponseEntity<>(successful, HttpStatus.OK);
+    }
+    return new ResponseEntity<>(successful, HttpStatus.NOT_FOUND);
+  }
+
+  @ApiMethod(description = "Add existing collections to an existing collection")
+  @PatchMapping(
+      value = {"/latest/collections/{uuid}/children", "/v3/collections/{uuid}/children"},
```

Comment:
Warum hier `children` und in `removeChild` dann auf einmal `child`?

Reply:
Hab den Pfad entsprechend angepasst.